packages/ai-jsx/src/batteries/constrained-output.tsx

```diff
@@ -78,9 +79,18 @@ export type TypedObjectCompletionWithRetry = TypedObjectCompletion & { retries?:
  *    Intermediate results that are valid, are also yielded.
```

Comment:
@farzadab can we update the JSDoc example for `JsonChatCompletion` to include the `schema` prop? Right now the `FamilyTree` var is not used.